calypsoIframe.js

```diff
@@ -0,0 +1,26 @@
+document.addEventListener("DOMContentLoaded", function() {
+  const container = document.getElementById("calypso");
+  const url = container.dataset.url;
+
+  const iframe = document.createElement("iframe");
+  iframe.setAttribute("src", url);
+  iframe.setAttribute("id", "myIframe");
+  iframe.style.width = "100%";
+  iframe.style.border = "none";
+
+  container.innerHTML = iframe.outerHTML;
+
+  // Add event listener for messages being massed from the iframe
+  const setHeight = e => {
+    // Check that message being passed is the documentHeight
+    if (typeof e.data === "string" && e.data.indexOf("documentHeight") > -1) {
```

Comment:
You should document that the iframe is waiting for a message in the format `documentHeight:20px`. Actually you could also pass that syntax as a parameter 😄

Reply:
What do you mean ?

Reply:
this script is waiting for a `message` (in the sense of `postMessage` API) that is of this format. This message is parsed in this script to extract the height.